CONTRIBUTING.md

```diff
@@ -70,58 +70,82 @@ GitHub provides additional documentation on [forking a repository](https://help.
 
 ## Debugging/Running Locally
 
-To test your changes locally, you can run the project from IntelliJ or gradle.
+To test your changes locally, you can run the project from IntelliJ or Gradle using the `runIde` tasks. Each build will download the required IDE version and 
+start it in a sandbox (isolated) configuration.
+
+### In IDE Approach (Recommended)
+
+Launch the IDE through your IntelliJ instance using the provided run configurations. 
+If ran using the Debug feature, a debugger will be auto-attached to the sandbox IDE.
+
+### Running manually
 
-- **Simple approach:** from the top-level of the repository, run:
-  ```
-  ./gradlew runIde --info
-  ```
-  The `runIde` task automatically downloads the correct version of IntelliJ
-  Community Edition, builds and installs the plugin, and starts a _new_
-  instance of IntelliJ with the built extension.
-- To run **Rider or "Ultimate"**, specify the respective gradle target:
   ```
+  ./gradlew jetbrains-core:runIde
   ./gradlew jetbrains-ultimate:runIde
   ./gradlew jetbrains-rider:runIde
   ```
   - These targets download the required IDE for testing.
-  - Do not specify `ALTERNATIVE_IDE`.
+
+#### Alternative IDE
+
 - To run the plugin in a **specific JetBrains IDE** (and you have it installed), specify the `ALTERNATIVE_IDE` environment variable:
   ```
-  ALTERNATIVE_IDE=/path/to/ide ./gradlew :runIde
+  ALTERNATIVE_IDE=/path/to/ide ./gradlew :intellij:runIde
```

Comment:
isn't this now a project property

Reply:
No, the profile is, but alt ide is still env var
```
    val alternativeIde = providers.environmentVariable("ALTERNATIVE_IDE").forUseAtConfigurationTime()
```